src/main/java/dev/blaauwendraad/masker/json/config/KeyMaskingConfig.java

```diff
@@ -0,0 +1,377 @@
+package dev.blaauwendraad.masker.json.config;
+
+import javax.annotation.CheckForNull;
+import java.nio.charset.StandardCharsets;
+import java.util.Objects;
+
+public class KeyMaskingConfig {
```

Comment:
🤔 
<img width="680" alt="image" src="https://github.com/Breus/json-masker/assets/15277543/0cf2998e-33a1-4754-b376-e20dda960799">

Reply:
But yeah, both of them can be final 😃